ajax/libs/cplayer/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "cPlayer",
+  "version": "2.3.0",
+  "description": "A beautiful and clean WEB Music Player by HTML5.",
+  "keywords": [
+    "cPlayer",
+    "html5",
+    "audio",
+    "music",
+    "player",
+    "es6",
+    "cplayer"
+  ],
+  "author":"Corps",
+  "filename":"cplayer.min.js",
+  "license": "MIT",
+  "repository": {
+    "type":"git",
+    "url":"git+https://github.com/Copay/cPlayer.git"
+  },
+  "autoupdate":{
+    "source":"git",
+    "target":"git://github.com/Copay/cPlayer.git",
+    "basePath":"dist",
+    "files":["**/*"]
+  }
```

Comment:
Let's follow the format in [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md).
```json
"autoupdate": {
    "source": "git",
    "target": "git://github.com/Copay/cPlayer.git",
    "basePath": "dist",
    "files": [
      "**/*"
    ]
}
```